ansible/group_vars/all.yml

```diff
@@ -77,7 +77,15 @@ shiftstack_local_storage_node_selector: beta.kubernetes.io/instance-type=nvme-8
 shiftstack_local_storage_device: /dev/nvme0n1
 shiftstack_thin_volume_count: 6
 shiftstack_thin_volume_size: 140g
-  
+
+################################################################################
+# Configure OpenShift Monitoring playbook vars
+################################################################################
+openshift_prometheus_retention_period: 30d
+openshift_prometheus_node_selector: "localstorage: true"
+openshift_prometheus_storage_class: local-sc
+openshift_prometheus_storage_size: 50Gi
```

Comment:
Just for the record, with local-storage I see the entire PV being used irrespective of what is specified here.